app/views/dashboard/assessmentDetails.scala.html

```diff
@@ -120,13 +140,13 @@
 <div id="details-content-valuations" class="govuk-body">
 
     @if(model.currentAssessments.nonEmpty){
-    <h2 class="govuk-heading-m">@messages("assessments.current.title")</h2>
-        @table(model.currentAssessments, "currentAssessments")
+        <h2 class="govuk-heading-m">@messages("assessments.current.title")</h2>
+        @table(model.currentAssessments, listType = "currentAssessments", isDraft = false)
```

Comment:
I don't understand where does this "currentAssessments" come from? That's not a list type...

Reply:
explained offline

Reply:
agreed approach